preprod.gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json

```diff
@@ -16,7 +16,7 @@
     "revproxy": "quay.io/cdis/nginx:2020.08",
     "sheepdog": "quay.io/cdis/sheepdog:2020.08",
     "metadata": "quay.io/cdis/metadata-service:2020.08",
-    "portal": "quay.io/cdis/data-portal:2020.08",
+    "portal": "quay.io/cdis/data-portal:2.33.0",
```

Comment:
We are going to roll 2020.09 into bdcat preprod soon @mpingram .
It seems like this change is not in place.
https://github.com/uc-cdis/data-portal/releases/tag/2020.09

Should we wait for 2020.10 that will be published in 2 weeks?